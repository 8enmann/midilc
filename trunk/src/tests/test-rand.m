/**
 * Test the random function. This test has to fail to be valid.
 * @author Ben
 */

main() {
  Note a; 
  Sequence b;
  Number i;
  Number r;
  b = new_sequence();
  a = As;
  for (i = 0 ; i < 20 ; i = i + 1) {
    r = rand(15);
    b = b + (a .+ r + r);
  }
  play(b);
}
