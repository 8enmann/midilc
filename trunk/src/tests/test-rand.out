0,4,78
4,4,81
8,4,78
12,4,72
16,4,72
20,4,79
24,4,79
28,4,76
32,4,79
36,4,81
40,4,80
44,4,74
48,4,71
52,4,78
56,4,76
60,4,74
64,4,69
68,4,82
72,4,81
76,4,82

